docs/docusaurus/versioned_docs/version-1.7.0/feg/s1ap_federated_test.md

```diff
@@ -190,7 +190,11 @@ vagrant ssh magma_test
 
 # inside vagrant vm
 cd magma/lte/gateway/python/integ_tests
-make integ_test TESTS=federated_tests/s1aptests/test_attach_detach.py
+## Run single test
```

Comment:
I don't think this change should be added to the v1.7 docs as this is a new test, i.e. one that wasn't used in v1.7, right?